sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -425,7 +425,7 @@ abstract class Element<T extends Widget> implements BuildContext {
   T get widget => _widget;
   T _widget;
 
-  RenderObject get _descendantRenderObject {
+  RenderObject get renderer {
```

Comment:
It's confusing to have both `renderer` and `renderObject`...  Maybe call this `renderObject` because the one in `RenderObjectElement` is essentially an override.